### PR TITLE
feat: update configs

### PR DESCRIPTION
**Pull request by zainab-amir**
### Description

- Updated the structure of configs
- Removed unused config `ENABLE_COPPA_COMPLIANCE`, `SHOW_DYNAMIC_PROFILING_PAGE` and `AUTHN_MINIMAL_HEADER`
- Replaced `COOKIE_DOMAIN` with `SESSION_COOKIE_DOMAIN` [reference edx-internal [common configs](https://github.com/edx/edx-internal/blob/4b3d365199d7bcf829dc0ea7939310d50742396c/frontends/common/prod_config.yml#L78)]
- Added a .env.private.example file

#### Merge Checklist

* [ ] If your update includes visual changes, have they been reviewed by a designer? Send them a link to the Sandbox, if applicable.
* [ ] Is there adequate test coverage for your changes?

#### Post-merge Checklist

* [ ] Deploy the changes to prod after verifying on stage or ask **@openedx/vanguards** to do it. 
* [ ] 🎉 🙌 Celebrate! Thanks for your contribution.

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -18,22 +18,18 @@ ORDER_HISTORY_URL='http://localhost:1996/orders'
 REFRESH_ACCESS_TOKEN_ENDPOINT='http://localhost:18000/login_refresh'
 SEGMENT_KEY=''
 SITE_NAME='Your Platform Name Here'
+INFO_EMAIL='info@example.com'
+# ***** Cookies
```
Comment (shafqatfarhan) [inline .env.development:22]:
```suggestion
# ***** Cookies *****
```

File: .env.development [hunk 2]
```diff
@@ -18,22 +18,18 @@ ORDER_HISTORY_URL='http://localhost:1996/orders'
...
+REGISTER_CONVERSION_COOKIE_NAME='openedx-user-register-conversion'
+SESSION_COOKIE_DOMAIN='localhost'
 USER_INFO_COOKIE_NAME='edx-user-info'
-AUTHN_MINIMAL_HEADER=true
-LOGIN_ISSUE_SUPPORT_LINK='/login-issue-support-url'
+USER_SURVEY_COOKIE_NAME='openedx-user-survey-type'
+# ***** Links
```
Comment (shafqatfarhan) [inline .env.development:27]:
```suggestion
# ***** Links *****
```

File: .env.private.example [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Copy these to the .env.private to enable edx specific functionality on local system
```
Comment (shafqatfarhan) [inline .env.private.example:1]: nit: edX

**Review by shafqatfarhan — APPROVED** [review]
Added a couple of suggestions, otherwise, looks good.

**Comment by arbrandes** [comment]
@zainab-amir, can we also update [the corresponding REAME section](https://github.com/openedx/frontend-app-authn#environment-variablessetup-notes) accordingly, please?

**Comment by zainab-amir** [comment]
@arbrandes sure. I will update it.

**Pull request merged**